cmd/lk/replay.go

```diff
@@ -0,0 +1,223 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"os"
+
+	"github.com/olekukonko/tablewriter"
+	"github.com/twitchtv/twirp"
+	"github.com/urfave/cli/v3"
+
+	"github.com/livekit/protocol/auth"
+	"github.com/livekit/protocol/replay"
+	lksdk "github.com/livekit/server-sdk-go/v2"
+)
+
+var (
+	ReplayCommands = []*cli.Command{
+		{
+			Name:            "replay",
+			Usage:           "experimental (not yet available)",
+			Category:        "I/O",
+			Hidden:          true,
+			HideHelpCommand: true,
+			Commands: []*cli.Command{
+				{
+					Name:   "list",
+					Before: createReplayClient,
+					Action: listReplays,
+				},
+				{
+					Name:   "load",
+					Before: createReplayClient,
+					Action: loadReplay,
+					Flags: []cli.Flag{
+						&cli.StringFlag{
+							Name:     "id",
+							Usage:    "replay ID",
```

Comment:
For all named flags, you can add backticks around a capitalized string to automatically use it as the arg name in Usage. We're also using Sentence case for the usage strings now, just for legibility.

```suggestion
							Usage:    "Replay `ID`",
```